app/views/constructions/_form.html.erb

```diff
@@ -65,7 +65,7 @@
         Jede Baustelle kann in einer oder mehreren Kategorien abgelegt werden.
       <p>
 
-      <%= render partial: "shared/partials/categories_form", locals: { record: construction, record_type: "construction", form: f } %>
+      <%= render partial: "shared/partials/categories_form", locals: { record: construction, record_type: "construction", form: f, tag_filter: "generic_item_construction_site" } %>
```

Comment:
maybe we should update the `record_type` here to `construction_site` as well to be in place with all the other base resources?